src/librustc/diagnostics.rs

```diff
@@ -112,6 +112,11 @@ reference when using guards or refactor the entire expression, perhaps by
 putting the condition inside the body of the arm.
 "##,
 
+E0018: r##"
+The value of static and const variables must be known at the compile-time which is
+not possible with a pointer or with a value pointed by a pointer.
```

Comment:
"a value referenced by a pointer" is probably closer to correct.  (Though I would need to review where this error message is used to know if that is actually the intention.)

Valid:
* "P points at X" or "P points to X"
* "P references X"

Invalid:
* "P points X" --- this is wrong (unless P is a mason/bricklayer and X is a chimney -- that's a joke).